### DISCUSSION TITLE
ODBC Driver : Test connection Button

### DISCUSSION THREAD

**Discussion by simonaubertbd** [duckdb/duckdb#5053] [Ideas]
### What happens?

I would like a "Test connection" button to test configuration

### To Reproduce

Open odbc driver : no button to test

### OS:

Windows 10

### DuckDB Version:

0.5.1

### DuckDB Client:

ODBC

### Full Name:

Simon AUBERT

### Affiliation:

Business Decision (Orange Business Services)

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by hannes** [comment]
Happy to review a PR

**Comment by simonaubertbd** [comment]
Hello @hannes . I'm not sure to understand your answer. Could you please clarify?

**Comment by hannes** [comment]
Certainly, if you would like this feature to be added, you are welcome to implement it yourself and send us a Pull Request so it can possibly get merged into DuckDB.

**Comment by simonaubertbd** [comment]
Thanks for the explanations, @hannes Sadly, I don't have the skills for that, that's why I made this suggestion as it would ease user life ;)

Same for my other idea.

Best regards,

Simon
